joomla/joomla-cms
# [4.0] Improve updater language strings

### Summary of Changes
(Hopefully) Improve the updater checkbox texts to be less clumsy. 


### Testing Instructions
Use the nightly XML https://update.joomla.org/core/nightlies/next_patch_list.xml to get an update and see the checkbox texts:

![grafik](https://user-images.githubusercontent.com/1229869/136290079-7212a5d3-0bd9-49bd-b94b-26db6d17bbd8.png)

![grafik](https://user-images.githubusercontent.com/1229869/136290109-d6caabde-482d-4adf-997a-8119988ff38e.png)
